src/Appwrite/Utopia/Response/Model/SyncExecution.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Appwrite\Utopia\Response\Model;
+
+use Appwrite\Utopia\Response;
+use Appwrite\Utopia\Response\Model;
+
+class SyncExecution extends Model
+{
+    public function __construct()
+    {
+        $this
+            ->addRule('status', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Execution Status.',
+                'default' => '',
+                'example' => 'completed',
+            ])
```

Comment:
Just a question that crossed my mind, can function throw a specific error to set this status code? It would be an awesome feature if we want to allow people to make "API servers" using our cloud functions.